docs/api/extending_the_rest_api.md

```diff
@@ -155,9 +164,29 @@ The easiest way to handle cache is to re-use the `CachedValue` Value Object. It
 When you want the response to be cached, return an instance of CachedValue, with your Value Object as the argument. You can also pass a location id using the second argument, so that the Response is tagged with it:
 
 ```
-return new CachedValue($helloValue, ['locationId', 42]);
+use eZ\Publish\Core\REST\Server\Values\CachedValue;
+//...
+    public function sayHello( $name )
+    {
+        return new CachedValue(
+          new HelloValue( $name ),
+          ['locationId'=> 42]
+        );
+
+    }
+
+```
+Below you find the correspondent response header when using varnish as reverse proxy:
+```
+Age →30
+Cache-Control →public, s-maxage=60
+Via →1.1 varnish-v4
+X-Cache →HIT
+X-Cache-Hits →2
+X-Location-Id →42
```

Comment:
In 1.12 it will be:
```
 +Age →30
 +Cache-Control →private, no-cache
 +Via →1.1 varnish-v4
 +X-Cache →HIT
 +X-Cache-Hits →2
```

So it's better if you show what you get when browsing the raw backend, casue as of 1.12 we change the output both in VCL for Varnish and in Symfoy proxy. See: https://github.com/ezsystems/ezplatform/blob/master/doc/varnish/vcl/varnish4_xkey.vcl#L225-L258